package.json

```diff
@@ -62,7 +62,7 @@
     "express": "^4.14.1",
     "express-nunjucks": "^2.2.3",
     "express-session": "^1.13.0",
-    "govuk-elements-sass": "3.1.2",
+    "govuk-elements-sass": "3.1.3",
```

Comment:
@shepda, we should probably look at including the characters that mean this auto updates to sub versions but won't upgrade to a main (possible breaking) version.  I think it's the `^` character or such.

Reply:
Yes it is ^.
Tbh I can't even remember making this change, maybe @adamb-moj told me about it? Not sure...